README.md

```diff
@@ -1,3 +1,8 @@
-# deskpro-styles
+deskpro-components
```

Comment:
The repository is named deskpro-components but the npm package is deskpro-styles
@chroder should we keep only one name?